internal/metrics/metrics.go

```diff
@@ -218,6 +218,7 @@ func requestDurationMetric() prometheus.HistogramVec {
 }
 
 func certificateExpiryDate(certString string) time.Time {
+	// TODO: Does this return the expiry date of the issuer?
```

Comment:
I like avoiding TODO comments. I'm not sure what's the goal of this one here anyway.
If we are passing the chain to the function, I think this function would return the expiry date of the first cert in the chain.
Should we create a bug or fix it here instead of keeping the TODO?

Reply:
I am not sure if it is a bug

Reply:
actually we already test this in the metrics test and it works. I'm removing it.